package.json

```diff
@@ -16,6 +16,7 @@
     "@emotion/styled": "^10.0.27",
     "@svgr/webpack": "^5.4.0",
     "@wordpress/api-fetch": "^3.19.0",
+    "@worldresources/gfw-components": "^3.5.4",
```

Comment:
I bumped this version some days ago, its ok, we will just need to do a rebase to the epic branch once you merge this PR 👍